docs/source/user-guide/guide_api.rst

```diff
@@ -35,42 +24,104 @@ or by running this drush command, passing in the dataset ID:
 
     .. code-block::
 
-      drush dkan:dataset-info [datasetID]
+      drush dkan:dataset-info {datasetID}
 
+In the following examples, we will use {datasetID} to represent the dataset identifier, {distributionID} to
+represent the distribution identifier, and {index} to represent which distribution is being referenced on a
+dataset.
 
-How to set the moderation state through the API.
-------------------------------------------------
+Anonymous API Examples
```

Comment:
I don't think the extra nesting is needed, listing authenticated examples below the blurb about authenticating should be enough.